src/main/scala/org/zalando/spearheads/innkeeper/dao/PathsPostgresRepo.scala

```diff
@@ -76,38 +76,67 @@ class PathsPostgresRepo @Inject() (
   override def collisionExistsForPath(path: PathIn): Future[Boolean] = {
     logger.debug("path uri collision check")
 
-    def position = SimpleExpression.binary[String, String, Int] {
+    val query = Paths
+      .filter(filterPathsWithUriCollision(path.uri, path.hasStar.contains(true)))
+      .map(_.hostIds)
+
+    db.run {
+      query.result
+    }.map{ existingPathHostIds =>
+      hostIdCollisionExists(path.hostIds, existingPathHostIds)
+    }
+  }
+
+  override def collisionExistsForPatch(pathId: Long, newHostIds: Seq[Long]): Future[Boolean] = {
+    logger.debug("path uri collision check")
+
+    selectById(pathId).flatMap {
+      case None => Future(true)
```

Comment:
If we can't select the path by Id, we return `true`, meaning a collision exists? I don't understand that. Seems wrong to me